coverage.py

```diff
@@ -133,9 +135,13 @@ def create_overdrive_api(self, overdrive_api_class):
             self._db, self.DEFAULT_OVERDRIVE_COLLECTION_NAME,
             ExternalIntegration.OVERDRIVE
         )
-        if is_new:
-            raise ValueError('Default Overdrive collection has not been configured.')
-        return overdrive_api_class(self._db, collection)
+        try:
+            return overdrive_api_class(self._db, collection)
+        except CannotLoadConfiguration, e:
+            self.log.error(
+                'Default Overdrive collection is not properly configured. No Overdrive work will be done.'
+            )
+            return
```

Comment:
This is an unrelated change I made so that I could run coverage providers without configuring an Overdrive integration.